newlib/libc/sys/arc/syscalls.c

```diff
@@ -35,6 +37,10 @@ _syscall2 (int,gettimeofday, struct timeval *,tv, void *,tz)
 
 _syscall1 (void,exit, int,ret)
 
+_syscall1 (_CLOCK_T_,times, struct tms *,buf)
```

Comment:
Indeed there are GNU style issues with this file (and not only). Please make the reformatting/cleaning up in a separate commit.